tools/synch.py

```diff
@@ -46,60 +46,8 @@
 # Code that does have a mirror in the mbed SDK
 # Tuple data: (repo_name, list_of_code_dirs, [team])
 # team is optional - if not specified, the code is published under mbed_official
-OFFICIAL_CODE = (
-    ("mbed-dev" , [MBED_DRIVERS, MBED_PLATFORM, MBED_HAL]),
-    ("mbed-rtos", RTOS),
```

Comment:
Oh and no @0xc0170 wont use the synch.py any more , only used during releasing and I do most of that now :)